canonical-web-and-design/jaas-dashboard
# Show the controller version in the controller view

## Done

The controller version wasn't shown when connected to a self-bootstrapped Juju controller. This adds that data.

## QA

- Pull code
- Have the Juju 2.8 snap installed and bootstrap a new controller on your choice of cloud.
- Follow the setup instructions for a [Juju qa](https://github.com/canonical-web-and-design/jaas-dashboard#connecting-a-local-dashboard-to-a-remote-controller)
- See the controller page for the version.

## Details

Fixes #554 

## Screenshots

![Screen Shot 2020-06-30 at 1 50 10 PM](https://user-images.githubusercontent.com/532033/86170961-2a62a980-bad9-11ea-93ed-22753de61ab0.png)
